client/v2/client.go

```diff
@@ -18,7 +20,7 @@ import (
 // UDPPayloadSize is a reasonable default payload size for UDP packets that
 // could be travelling over the internet.
 const (
-	UDPPayloadSize = 512
+	UDPPayloadSize = 1500
```

Comment:
1500 is too almost certainly too much for the UDP payload, and is generally the largest standard MTU for ethernet frames. You'd have to account for the headers for ethernet, IP(4 and 6), and UDP.

IPv4 only guarantees 576, and MTU discovery would have to be done before larget packets could be guaranteed not to fragment.

512 is the recommended "safe" payload size to avoid fragmentation. The default can be overridden with the config if you know the path between your client and the influxdb instance can handle larger frames